src/engraving/dom/segment.cpp

```diff
@@ -2624,6 +2633,28 @@ double Segment::spacing() const
     return m_spacing;
 }
 
+bool Segment::isFullMeasureRestOrMMRestSegment() const
```

Comment:
I can't find where this is used, but if it's only needed for layout, it should be moved to the corresponding class

Reply:
Yep, forgot to remove it